permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -23,6 +25,54 @@ pub static ACCOUNT_ID_TOKEN_PARAM_NAME: Lazy<Name> =
 pub static ASSET_DEFINITION_ID_TOKEN_PARAM_NAME: Lazy<Name> =
```

Comment:
we should get rid of these?

Reply:
We totally should. My plan was to use [these](https://github.com/hyperledger/iroha/pull/2156/files/91181dac33ddbdbe84db4f54141ea3002c62a91c#diff-d2c070e02dba4b9ceb79b2f4ff86d6b8e15f498f20fa00d916c7259aa999d896R133) methods to replace them where they're still used, that's why they return `&'static Name` and not actual parameter values.